package.json

```diff
@@ -36,6 +36,7 @@
     "set-middleware": "set-middleware",
     "set-dev-endpoints": "node scripts/set-dev-endpoints.js",
     "postinstall": "npm run build",
+    "release:changelog": "echo -e \"$(head -3 CHANGELOG.md)\n\n$(node_modules/.bin/conventional-changelog -p angular)\n\n$(tail -n +3 CHANGELOG.md)\" > CHANGELOG.md",
     "release": "standard-version"
```

Comment:
this should probably be
`"release:standard-version": "standard-version -- --no-verify"`
then have release call both
"release": "release:standard-version && release:changelog"

Reply:
Thank you, yes this works! updated